### PR TITLE
[ui] only show uniq Contests based on name

### PR DESCRIPTION
**Pull request by teaforthecat**
This is a shim to make the ui look like it fits the new model -> Each Contest Name must be unique.

### CODE REVIEW

File: client/src/component/DOS/DefineAudit/SelectContestsForm.tsx [hunk 1]
```diff
@@ -207,8 +219,9 @@ class SelectContestsForm extends React.Component<FormProps, FormState> {
 
         };
         const filteredData = _.filter(sortedData, filterFunc);
+        const uniqData = _.filter(filteredData, uniqByName);
```
Comment (gws) [inline client/src/component/DOS/DefineAudit/SelectContestsForm.tsx]: Is there a way to use `_.uniqBy` here instead?

Comment (teaforthecat) [reply client/src/component/DOS/DefineAudit/SelectContestsForm.tsx]: I tried but I couldn't get the types to agree. Suggestions welcome.

**Comment by gws** [comment]
@teaforthecat Can you test with the data you were using with the commit I just made?

**Comment by teaforthecat** [comment]
@gws I know, it seems like that should work, but it doesn't do the job. I'm not sure why. uniqBy nor uniqWith worked for me. Perhaps because of string/object equality? idk.

**Comment by gws** [comment]
@teaforthecat I have one more thing to fix - the contest with reported ties component needs to have one less `<td>` as well or you get some weird lookin' tables!

I'll fix that in the base branch.

**Pull request merged**